### DISCUSSION TITLE
[Projects Beta] Feedback on View Layout

### DISCUSSION THREAD

**Discussion by dcmbrown** [community/community#6593] [Projects and Issues]
So as part of an organization the one biggest issue with the new project board which I noticed right away was that there is no easy way to get back to navigation of the organization. 
Old Setup (doctored screenshot):
![Screen Shot 2021-10-21 at 11 18 07 AM-RandomOrg](https://user-images.githubusercontent.com/569191/138328546-26e9509e-fea5-424c-bbe9-8b22f2419376.png)

vs the new setup:
![Screen Shot 2021-10-21 at 11 17 45 AM](https://user-images.githubusercontent.com/569191/138328623-552bc712-09d1-4522-9129-643213110e93.png)

Sure you can click into a project, or a task/issue and go to the organization from there but there is definitely a disconnect in the UX not being able to navigate back to another part of your organization without going to github root (github.com/) first, your personal account, and then to your organization.  Analytics may say these menu items are in fact used very little but it's not about the actual usage so much as  the project beta boards seem entirely separate with (currently) allowing no branding or other navigation clues to its place on github within your personal account or organization.

**Comment by mkungla** [comment]
Collected all related discussions here: https://github.com/github/feedback/discussions/10448
